addons/medical_status/functions/fnc_setUnconsciousState.sqf

```diff
@@ -46,6 +46,12 @@ if (_active) then {
             closeDialog 0;
         };
     };
+    // Unlock controls for copilot if unit is pilot of aircraft
+    if (vehicle _unit isKindOf "Air" && {_unit == driver vehicle _unit}) then {
```

Comment:
I think, since you do the Air check first, you can use as second check: _unit == driver _unit as well. (saves 1 command :P)

Reply:
![image](https://user-images.githubusercontent.com/9376747/74765289-e2896700-5248-11ea-8cbd-e15849150cbd.png)

Reply:
ok :(